pkg/releasing/defs.bzl

```diff
@@ -15,6 +15,7 @@ def print_rel_notes(
     print_rel_notes_helper = Label("//releasing:print_rel_notes")
     tools = [print_rel_notes_helper]
     cmd = [
+        "exec env LC_ALL=C.UTF-8",
```

Comment:
That does not seem like it would work right.  The exec will end the shell.
How about 
`"LC_ALL=C.UTF-8 $(location %s) ....  `

That should set the variable for the invocation of the process.